.github/workflows/check.yml

```
@@ -20,44 +20,37 @@ jobs:
+    strategy:
+      matrix:
+        package_path:
+          - packages/app
+          - packages/common
+          - packages/data
+          - packages/dependency_provider
+          - packages/domain
+
     steps:
       - name: Checkout
         uses: actions/checkout@v3
```

actionlint (1.7.4)

the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)